pyrasterframes/src/main/python/docs/aggregation.pymd

```diff
@@ -25,38 +26,39 @@ UNION
 SELECT 2 as id, rf_local_multiply(rf_make_ones_tile(5, 5, 'float32'), 3) as tile
 """)
 
-rf.select("id", rf_render_matrix("tile")).show(10, False)
+rf.select("id", rf_render_matrix("tile")).show(truncate=False)
```

Comment:
sort `rf` by id please. In my local run i get row 2 first, which is a bit disorienting

Reply:
Another thought, in this page it may be fine just to use the regular text rendering of the tile since the values are consistent in all cells. so `[[ 3, 3, 3, ... ] ]` would be fine

Reply:
we could also consider shrinking the tile dimension some